data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +517,1013 @@ mission "Unfettered Jump Drive Trading"
+		dialog `The Unfettered "pirate" spits a string of old-timey nautical curses, but one word, which you can barely hear him mutter, stands out: "Cerberus." He then comes closer to address you directly, "There be even more of these scurvy bilge rats than I expected. I warned ye about coming here with a jump drive!"`
+	npc evade
+		government "Cerberus"
+		system "Almaaz"
+		personality disables plunders vindictive
+		fleet "Large Northern Pirates"
+
+
+
+# This variant has no reverse thrusters to make it easier to scan it and to attract more attention,
+# especially with the keystones.
+ship "Geocoris" "Geocoris (Keystones)"
+	crew 56
+	outfits
+		"Hai Tracker Pod" 2
+		"Hai Tracker" 112
+		"Chameleon Anti-Missile" 3
+		"Hai Cuttlefish Jammer" 4
+		"Boulder Reactor"
+		"Hai Valley Batteries"
+		"Hai Diamond Regenerator"
+		"Hai Corundum Regenerator"
+		"Hai Williwaw Cooling" 4
+		`"Biroo" Atomic Thruster`
+		`"Biroo" Atomic Steering`
+		Hyperdrive
+		"Pulse Rifle" 56
+	cargo
+		outfits
+			"Quantum Keystone" 123
+
+mission "<uhai> 4: Scan Geocoris"
+	name `Scan the Geocoris`
+	description `Scan the cargo of the Hai Geocoris in Ya Hai with the Value Detector, then return to <destination> with the scanner logs. You may abort this, at a cost.`
+	source "Firelode"
+	waypoint "Ya Hai"
+	to offer
+		random < 60
+		has "event: unfettered: access to Mera's lab"
+	on offer
+		conversation
+			`As you wander through the spaceport, you find a market with a limited diversity of goods, although some of it seems to be completely alien, even by Hai standards. You can even spot human food and technology, although those are the only recent items here. Most of it is very ancient, and you doubt everything is functional...`
+			`	Most Unfettered here are just busy buying what little food there is, that is if they can even afford it. You see two of them disputing, probably over the cost of some item, but decide not to intervene given how they generally respond to your mere presence.`
+			`	An Unfettered approaches you. At first, you wonder why the Hai happens to be coming towards you in particular, but then you realize that you naturally stand out in the crowd. There are not many non-Hai on the Unfettered worlds.`
+			`	"Captain <last>! I am Mera, an engineer. We have an easy job for you, if you are interested," she says, and adds, "It would pay well."`
+			choice
+				`	"Sure, what can I do for you?"`
+				`	"How do you know me?"`
+					goto know
+				`	"Sorry, but I'm quite busy at the moment."`
+					defer
+			action
+				"unfettered rep: Mera" ++
+			`	"Straight to the point! I am starting to like you."`
+				goto job
+			label know
+			branch JD
+				has "Unfettered Jump Drive 1: offered"
+			`	"I know you have helped us when we needed it most. Word quickly got around of a human meddling in our business, but most view it in a positive way, such as myself. I could use some of that interference myself, in fact!" She says with a slight movement of the ears.`
```

Comment:
Why would most UHai consider a human meddling in their business a good thing? Perhaps elaborate on that here.

Reply:
prob should change "most" to "some"
you have been bringing some medical supplies, and the lab was opened to you caus the Chief from earlier told her about it